active/social-media.md

```diff
@@ -0,0 +1,81 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability through campaigns and sponsor relations.
+- Ensure the DSF’s Code of Conduct is maintained on its media channels.
+- Assist the board in posting official news and updates from the DSF.
+
+The group opts to manage platforms that best achieve these goals.
+This may be a subset of those for which Django has accounts.
+
+Delegated responsibilities:
+- The Chair can grant access to Django's official social media profiles and each member will have access.
+- Members can post content to Django's official social media profiles, in alignment with the working group's agreed 
+  goals, without a board review.
+- Members can perform actions such as “like”, “repost” or “comment” as Django, when it aligns with the working group's 
+  agreed goals, without a board review.
+- Members can perform maintenance and monitor the automations which post DSF news and updates to social media channels.
+- Members can enforce the Code of Conduct. This may include moderating comments that do not meet our CoC or blocking 
+  users that grossly or repeatedly violate the CoC.
+
+Actions to take back to the DSF Board for votes:
+- Amendments to the working group's goals.
+- Approval on content that is outside the agreed group's goals.
+- Proposals of social media campaigns for our sponsors.
+- Proposal to have a new social media profile on a platform.
+- Proposal to retire a social media profile from a platform.
+
+## Initial membership
+
+- Chair:
+- Co-Chair:
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair):
+- Other members:
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have demonstrable experience with social media platforms, and be registered as a Django Software Foundation
+Individual Member.
+
+### How do people who want to join sign up / volunteer / express interest?
+
+By sending an email to: `social-media-wg@djangoproject.com`
```

Comment:
Could this be done with Google Forms rather than email? The emails I receive from public DSF `foundation@` address are 80% spam. Using a form would also help in communicating what information we expect from new members.

If so, the form would be set up under the DSF’s Google Workspace, all entries would go in a spreadsheet, the group would review entries as they come in just like over email.

Reply:
Sure good point 👍 

Note: this (or something else) could become standard for all working groups. We can have a list of working groups and an application form for each on the website and the group only needs to define the questions. It will be easier for people to form working groups when we have some defined as "standard"